update_sample_status/main.py

```diff
@@ -1,63 +1,120 @@
-"""A Cloud Function to update the status of genomic samples."""
+"""A Cloud Function to update the status of genomic samples.
+Workflow is as follows
+1. gVCFs and CRAMS moved.
+2. Following successful move, analysis objects are created.
+3. Once analysis objects are successfully created, seqencing metadata is updated."""
 
-import json
-import os
 import logging
-from airtable import Airtable
-from flask import abort
-from google.cloud import secretmanager
-from requests.exceptions import HTTPError
+from os.path import join
+from typing import Dict
+from aiohttp import web
+from google.cloud import storage
+from sample_metadata.api.sample_api import SampleApi
+from sample_metadata.api.sequence_api import SequenceApi
+from sample_metadata.models.sequence_update_model import SequenceUpdateModel
+from sample_metadata.models import SequencingStatus
 
 
-secret_manager = secretmanager.SecretManagerServiceClient()
+routes = web.RouteTableDef()
 
+FILES_HANDLED = {
+    'gvcf': [{'primary': '.g.vcf.gz', 'secondary': ['.g.vcf.gz.tbi', '.g.vcf.gz.md5']}],
+    'cram': [{'primary': '.cram', 'secondary': ['.cram.crai', '.cram.md5']}],
+}
```

Comment:
To address future comment.

```suggestion
FILES_HANDLED = {
    'gvcf': [{'primary': '.g.vcf.gz', 'secondary': ['.g.vcf.gz.tbi', '.g.vcf.gz.md5'], 'meta-field': 'gvcf'}],
    'cram': [{'primary': '.cram', 'secondary': ['.cram.crai', '.cram.md5'], 'meta-field': 'reads'}],
}
```